extensions/git/src/repository.ts

```diff
@@ -626,6 +626,10 @@ export class Repository implements Disposable {
 		this.status();
 	}
 
+	updateInputBox() {
+		this._sourceControl.inputBox.placeholder = 'Message ({0} to commit to ' + (this._HEAD ? this._HEAD.name || '' : '') + ')';
```

Comment:
I'm pretty sure this breaks the localization of the string. See my comment above....

Reply:
@burknator Is right. Let's fix this.

Also, if `this._HEAD` doesn't exist, the message will become `Message ({0} to commit to )`. Let's fix that too.

Reply:
Fixed